custom/icds_reports/tasks.py

```diff
@@ -183,7 +183,7 @@ def run_move_ucr_data_into_aggregation_tables_task():
         move_ucr_data_into_aggregation_tables.delay(force_citus=True)
 
 
-@serial_task('move-ucr-data-into-aggregate-tables-{force_citus}', timeout=36 * 60 * 60, queue='icds_aggregation_queue')
+@serial_task('{force_citus}', timeout=36 * 60 * 60, queue='icds_aggregation_queue')
```

Comment:
did the key need to change or was it a stylistic choice? would the other format have worked

Reply:
stylistic choice - the name of the function is already pre-pended to the key here: https://github.com/dimagi/commcare-hq/blob/8a1f95117684ee4765b78aa0129ae360cf411214/corehq/util/decorators.py#L89

Reply:
I also looked at other usages of it and followed the pattern